src/main/scala/rocket/BTB.scala

```diff
@@ -16,7 +16,7 @@ case class BTBParams(
 
 trait HasBtbParameters extends HasCoreParameters {
   val btbParams = tileParams.btb.getOrElse(BTBParams(nEntries = 0))
-  val matchBits = pgIdxBits max log2Ceil(p(coreplex.CacheBlockBytes) * tileParams.icache.get.nSets)
+  val matchBits = pgIdxBits
```

Comment:
Now that I look at this again, I can't understand why it would've mattered.  In this configuration, pgIdxBits and log2Ceil(p(coreplex.CacheBlockBytes) * tileParams.icache.get.nSets) should be equal.